zombienet_tests/functional/0003-parachains-garbage-candidate.toml

```
@@ -0,0 +1,46 @@
+[settings]
+timeout = 1000
+bootnode = true
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+  max_validators_per_core = 2
```

I was struggling to understand why it works with 1 malus node. https://github.com/paritytech/polkadot/blob/4863d0a33a4a3534236f76abb5b1dc91751c6c34/runtime/parachains/src/inclusion/mod.rs#L589-L593
It does work because `group_vals.len()` here is 1 because we have 4 validators and 4 parachains. Setting `max_validators_per_core = 2` is a bit confusing, I'm suggesting setting it to 1.
```suggestion
  max_validators_per_core = 1
```